enterprise/utils.py

```diff
@@ -1227,3 +1238,34 @@ def delete_tableau_user_by_id(tableau_user_id):
             '[TABLEAU USER SYNC] Could not delete the tableau user %s.',
             tableau_user_id,
         )
+
+
+def unset_language_of_all_enterprise_learners(enterprise_customer):
```

Comment:
Is this part of the solution proposed in the JIRA ticket ENT-3522?

Reply:
No, during testing I found, I had to do these too in order for the solution to work correctly.